<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerManage</class>
 <widget class="QWidget" name="CustomerManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>788</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Client Info</string>
  </property>
  <property name="windowIcon">
   <iconset theme="applications-science">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QPushButton" name="addPushButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>370</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifyPushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Modify</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="rootIsDecorated">
    <bool>false</bool>
   </property>
   <property name="expandsOnDoubleClick">
    <bool>true</bool>
   </property>
   <attribute name="headerVisible">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Phone Number</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Email</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Domain</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Address</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Favorite</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Age</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Gender</string>
    </property>
   </column>
  </widget>
  <widget class="QTreeWidget" name="searchTreeWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>681</width>
     <height>151</height>
    </rect>
   </property>
   <property name="rootIsDecorated">
    <bool>false</bool>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Quantity</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="searchComboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>620</y>
     <width>140</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ID</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Name</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Phone Number</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Address</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="searchPushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>610</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>620</y>
     <width>210</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="favoriteComboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Medical</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Health</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Diet</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="ageLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Age</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="ageSpinBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>19</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="genderLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>45</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="genderGroupBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>25</height>
    </size>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="femaleButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="maleButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="favoriteLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Favorite</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="domainLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>@</string>
   </property>
  </widget>
  <widget class="QComboBox" name="domainComboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>126</width>
     <height>32</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>osstem.com</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>naver.com</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>yahoo.com</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>daum.net</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>gmail.com</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="addressLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Address</string>
   </property>
   <property name="buddy">
    <cstring>addressLineEdit</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="idLineEdit">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>21</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>52</y>
     <width>35</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Name</string>
   </property>
   <property name="buddy">
    <cstring>nameLineEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>21</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>I&amp;D</string>
   </property>
   <property name="buddy">
    <cstring>idLineEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="phoneNumberLabel">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>83</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Phone Number</string>
   </property>
   <property name="buddy">
    <cstring>phoneNumberLineEdit</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>52</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneNumberLineEdit">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>83</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="emailLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>32</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
